README.md

```diff
@@ -7,10 +7,10 @@ Tool for automatically upgrading D source code
 * Updates old-style alias syntax to new-style
 * Fixes implicit concatenation of string literals
 * Automatic conversion of C-style array declarations and parameters to D-style.
-* Upgrades code to comply with DIP64 when the ```--dip64``` switch is specified. (Not recommended)
-* Upgrades code to comply with DIP65 unless the ```--dip65=false``` switch is specified.
-* Upgrades code to comply with DIP1003 unless the ```--dip1003=false``` switch is specified.
-* Rewrites functions declared ```const```, ```immutable``` and ```inout``` to be more clear by moving these keywords from the left side of the return type to the right side of the parameter list.
+* Upgrades code to comply with [DIP64](https://wiki.dlang.org/DIP64) when the `--dip64` switch is specified. (Not recommended)
```

Comment:
for consistence you can use the link to the md files here https://github.com/dlang/DIPs/tree/master/DIPs/archive

Reply:
That archive doesn't list DIP64 or DIP65.